fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -355,7 +419,7 @@ private static void timerIncrement(@Nullable FDBStoreTimer timer, FDBStoreTimer.
 
     abstract CompletableFuture<Void> rebuildIndexByEntityAsync(FDBRecordStore store);
 
-    // These two methods are externalized for testing usage only
+    // These throttle methods are externalized for testing usage only
```

Comment:
I removed the `@VisibleForTesting` annotation from these functions because it caused compilation warnings (as being called by a non-testing function, although itself marked as `@VisableForTesting`).